docs/core/tutorials/netcore-hosting.md

```diff
@@ -19,40 +19,127 @@ Because hosts are native applications, this tutorial will cover constructing a C
 
-Keep in mind that the sample host is meant to be used for learning purposes, so it is light on error checking and is designed to emphasize readability over efficiency. More real-world host samples are available in the [dotnet/coreclr](https://github.com/dotnet/coreclr/tree/master/src/coreclr/hosts) repository. The [CoreRun host](https://github.com/dotnet/coreclr/tree/master/src/coreclr/hosts/corerun), in particular, is a good general-purpose host to study after reading through the simpler sample.
+Keep in mind that the sample hosts are meant to be used for learning purposes, so they are light on error checking and are designed to emphasize readability over efficiency. More real-world host samples are available in the [dotnet/coreclr](https://github.com/dotnet/coreclr/tree/master/src/coreclr/hosts) repository. The [CoreRun host](https://github.com/dotnet/coreclr/tree/master/src/coreclr/hosts/corerun) and [Unix CoreRun host](https://github.com/dotnet/coreclr/tree/master/src/coreclr/hosts/unixcorerun), in particular, are good general-purpose hosts to study after reading through these simpler samples.
+
+## Create a host using CoreClrHost.h
+
+The following steps detail how to use the CoreClrHost.h API to start the .NET Core runtime in a native application and call into a managed static method. The code snippets in this document use some Windows-specific APIs, but the [full sample host](https://github.com/dotnet/samples/tree/master/core/hosting/HostWithCoreClrHost)  shows both Windows and Linux code paths.
+
+### Step 1 - Find and load CoreCLR
+
+The .NET Core runtime APIs are in *coreclr.dll* (on Windows), in *libcoreclr.so* (on Linux), or in *libcoreclr.dylib* (on macOS). The first step to hosting .NET Core is to load the CoreCLR library. Some hosts probe different paths or use input parameters to find the library while others know to load it from a certain path (next to the host, for example, or from a machine-wide location).
+
+Once found, the library is loaded with `LoadLibraryEx` (on Windows) or `dlopen` (on Linux/Mac).
+
+[!code-cpp[CoreClrHost#1](../../../samples/core/hosting/HostWithCoreClrHost/SampleHost.cpp#1)]
+
+### Step 2 - Get .NET Core hosting functions
+
+CoreClrHost has several important methods useful for hosting .NET Core:
+
+* `coreclr_initialize`: Starts the .NET Core runtime and default AppDomain
+* `coreclr_execute_assembly`: Executes a managed assembly
+* `coreclr_create_delegate`: Creates a function pointer to a managed method
+* `coreclr_shutdown`: Unloads AppDomains and shuts down the .NET Core runtime
```

Comment:
Same as above - remove the AppDomain mention

Reply:
Removed